### ISSUE TITLE
Edit button not appearing

### ISSUE THREAD

**Issue by brandondurham** [NAlbuquerque/Category-Field-for-EE#1]
Any idea why my Edit button wouldn’t be appearing? I poked through the DOM and see a `.cat_action_buttons` DIV with `display: none` on it.

I’m interested in this add on because I want my client to be able to select existing categories OR add new easily.

**Comment by NAlbuquerque** [comment]
It essentially takes the edit link from the original category controls. Make sure the member group has rights to manage the category group in question.

**Comment by stovepot** [comment]
![Sequel Pro](https://f.cloud.github.com/assets/130454/27811/e95cb280-4ba7-11e2-978c-c5cfc563b754.png)
Having the same issue - logged in as Super Admin, but also other member groups, which have access to Categories (Granted Privileges). Only one anomaly\* which is that category_field 1.2 shows on both field type id line 45 & 46 with identical settings: YToxOntzOjE3OiJjYXRlZ29yeV9ncm91cF9pZCI7czowOiIiO30=

Note*: not that I know mysql much…..

**Comment by NAlbuquerque** [comment]
Can you try using the version in the Dev branch? That version fixes a bug relating to how the field settings are stored and might fix your issue as well.

**Comment by stovepot** [comment]
I just did but noticed no difference...

**Comment by NAlbuquerque** [comment]
So you have two instances of Category Field in your channel. And each has its own setting? Or do they have the same settings? Can you make sure you edit the field type and resave the settings for both.

**Comment by stovepot** [comment]
Well that's the thing - I have only one instance of Category Field in my Channel. Not sure where the second instance comes from, although I have been in-and uninstalling the Fieldtype to attempt to rid of the issue….

**Comment by NAlbuquerque** [comment]
Sounds like something went wonky while uninstalling the fieldtype. Can you please uninstall the fieldtype and check the exp_fieldtypes table again? Verify that the version is 1.4.

**Comment by stovepot** [comment]
Uninstalled the Fieldtype, cleared cache, reinstalled Fieldtype, checked version 1.4. Changed settings to "Select List", but still NO edit link…

Fieldtype showing only once in database…. Are there any other native settings I should check?

**Comment by NAlbuquerque** [comment]
Does the member have access to edit the category group you selected?
In the native categories publish tab, do you see an edit link?
Do you get any javascript errors?

**Comment by stovepot** [comment]
Added an additional Category Group (did only have one) but no difference. Mind you the native Category tab is empty despite all privileges...

**Comment by NAlbuquerque** [comment]
Ok I was able to reproduce the problem. It happens when there is only a single category group assigned to the channel. I'll issue an update soon!

**Comment by stovepot** [comment]
Not sure why I cannot solve it on my end with multiple category groups - weird is that the native Category Tab is empty...

**Comment by NAlbuquerque** [comment]
Ok, I believe this bug is now fixed. Please grab a new copy from the dev branch and let me know how it goes. If all goes well I'll merge into master branch!

**Comment by stovepot** [comment]
Thanks! All working nicely…
